src/test/java/uk/gov/hmcts/reform/sandl/snlevents/actions/hearingpart/AssignHearingPartToSessionActionTest.java

```diff
@@ -130,6 +130,12 @@ public void act_shouldSetClassFieldsToProperState() throws JsonProcessingExcepti
         Mockito.verify(objectMapper, times(2)).writeValueAsString(any());
     }
 
+    @Test(expected = SnlEventsException.class)
+    public void act_shouldThrowServiceException_whenHearingIsMultiSession() {
+        mockedHearingPart.getHearing().setMultiSession(true);
+        action.getAndValidateEntities();
```

Comment:
There are multiple SnlEventsException possible to be thrown in that method. Please verify the message as well

Reply:
Done! :)